RoosterPlanner.UI/src/app/services/project.service.ts

```diff
@@ -28,24 +29,31 @@ export class ProjectService {
   }
 
   postProject(project: Project) {
-    if(project==null){
+    if (project == null) {
       window.alert("Leeg project in project service");
```

Comment:
Oppassen met window.alert. Dit kan hinderlijk zijn voor de gebruiker. Gebruik hiervoor een toastr, dit is vriendelijker voor de gebruiker, want er is geen interactie mee nodig e.d.